src/main/kotlin/no/nav/syfo/client/aareg/domain/AaregArbeidsforholdOversikt.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.syfo.client.aareg.domain
+
+data class AaregArbeidsforholdOversikt(
+    val arbeidsforholdoversikter: List<Arbeidsforholdoversikt>,
+)
+
+data class Arbeidsforholdoversikt(
+    val arbeidssted: Arbeidssted,
+    val opplysningspliktig: Opplysningspliktig,
+)
+
+data class Arbeidssted(
+    val type: ArbeidsstedType,
+    val identer: List<Ident>
+) {
+    fun getOrgnummer() =
+        identer.firstOrNull {
+            it.type == IdentType.ORGANISASJONSNUMMER
+        }?.ident
+}
+
+data class Opplysningspliktig(
+    val type: OpplysningspliktigType,
+    val identer: List<Ident>,
+) {
+    fun getJuridiskOrgnummer() =
+        identer.firstOrNull {
+            it.type == IdentType.ORGANISASJONSNUMMER
+        }?.ident
+}
+
+data class Ident(
+    val type: IdentType,
+    val ident: String,
+    val gjeldende: Boolean,
+)
+
+enum class ArbeidsstedType {
+    Underenhet,
+    Person,
+}
+
+enum class OpplysningspliktigType {
+    Hovedenhet,
+    Person,
+}
+
+enum class IdentType {
+    AKTORID,
```

Comment:
Tenker vi å bruke `AKTORID` senere? 
Ser ike ut til at den er i bruk.

Reply:
Nei tror ikke det, ble bare bekymret for om det ville kunne feile dersom vi mangler verdien. Litt usikker på hvordan jackson håndterer det dersom deler av innholdet i enum ikke finnes. Men om det ikke er noe problem kan vi ta det bort (?)

Reply:
Fjernet og kjørte tester, så gikk det jo ok. Men har ikke testet i `dev` miljøet.

Reply:
Nei mest bekymret dersom det kommer inn JSON som har en liste som også inneholder verdier vi ikke har

Reply:
Det ser ut som det vil kræsje uten håndtering. Noen anbefaler å lage en fake enum value som default - @JsonEnumDefaultValue UNKNOWN;, og så i objectmapper setter man: ObjectMapper().enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE);

Reply:
Da er det beste som du gjør da :) 👍🏼